macOS/DuckDuckGo-macOS.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -77,8 +77,8 @@
       "kind" : "remoteSourceControl",
       "location" : "https://github.com/duckduckgo/privacy-dashboard",
       "state" : {
-        "revision" : "fd3b7e2d8194e265c537f227c706de7db3085da5",
-        "version" : "9.4.0"
+        "revision" : "92c9c3970ddc6062fa2952e05098fdaea499ab6a",
+        "version" : "9.5.0"
```

Comment:
Thanks for including it - we could probably benefit from a Danger check to warn when files are modified at the end of a build, to remind people that when they update a dependency they should also check in Package.resolved.